.amethyst.sample.yml

```
@@ -1,42 +1,280 @@
+
+# Expand the main pane by a percentage of the screen dimension as defined by window-resize-step. Note that not all layouts respond to this command.
+expand-main:
+  mod: mod1
+  key: l
+
+# Increase the number of windows in the main pane. Note that not all layouts respond to this command.
+increase-main:
+  mod: mod1
+  key: ","
+
+# Decrease the number of windows in the main pane. Note that not all layouts respond to this command.
+decrease-main:
+  mod: mod1
+  key: "."
+
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command1:
+#   mod: <NONE>
+#   key: <NONE>
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command2:
+#   mod: <NONE>
+#   key: <NONE>
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command3:
+#   mod: <NONE>
+#   key: <NONE>
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command4:
+#   mod: <NONE>
+#   key: <NONE>
+
+# Focus the next window in the list going counter-clockwise.
+focus-ccw:
+  mod: mod1
+  key: j
+
+# Focus the next window in the list going clockwise.
+focus-cw:
+  mod: mod1
+  key: k
+
+# Focus the main window in the list.
+focus-main:
+  mod: mod1
+  key: m
+
+# Focus the next screen in the list going counter-clockwise.
+focus-screen-ccw:
+  mod: mod1
+  key: p
+
+# Focus the next screen in the list going clockwise.
+focus-screen-cw:
+  mod: mod1
+  key: n
+
+# Move the currently focused window onto the next screen in the list going counter-clockwise.
+swap-screen-ccw:
+  mod: mod2
+  key: h
+
+# Move the currently focused window onto the next screen in the list going clockwise.
+swap-screen-cw:
+  mod: mod1
+  key: l
+
+# Swap the position of the currently focused window with the next window in the list going counter-clockwise.
+swap-ccw:
+  mod: mod2
+  key: j
+
+# Swap the position of the currently focused window with the next window in the list going clockwise.
+swap-cw:
+  mod: mod2
+  key: k
+
+# Swap the position of the currently focused window with the main window in the list.
+swap-main:
+  mod: mod1
+  key: "enter"
+
+# Move focus to the n-th screen in the list; e.g., focus-screen-3 will move mouse focus to the 3rd screen. Note that the main window in the given screen will be focused.
+#focus-screen-n:
+# focus-screen-<screen-number>:
+#   mod: mod1
+#   key: y
+# Move the currently focused window to the n-th screen; e.g., throw-screen-3 will move the window to the 3rd screen.
+# throw-screen-n:
+# throw-screen-<screen-number>:
+#   mod: mod1
+#   key: u
+# Move the currently focused window to the n-th space; e.g., throw-space-3 will move the window to the 3rd space.
+# throw-space-<screen-number>:
+#   mod: mod1
+#   key: i
+
+# Move the currently focused window to the space to the left.
+throw-space-left:
+  mod: mod2
+  key: "left"
+
+# Move currently the focused window to the space to the right.
+throw-space-right:
+  mod: mod2
+  key: "right"
```

Based on suggestion above
```suggestion
  key: right
```